configs/default/FRXS-FREEAXISF4.config

```
@@ -0,0 +1,122 @@
+# Betaflight / STM32F411 (S411) 4.1.0 Jun 25 2019 / 10:27:57 (2a6e94d03) MSP API: 1.42
+
+#mcu STM32F411
+
+#define USE_GYRO
+#define USE_ACC
+#define USE_ACCGYRO_BMI270
+#define USE_FLASH
+#define USE_FLASH_W25N01G
+#define USE_MAX7456
+
+board_name FREEAXISF4
+manufacturer_id FRXS
```

```suggestion
board_name MATEKF411_CLONE
manufacturer_id CLNE
```
Before submitting new targets we need a manufacturer. As this board again uses another manufacturers work, rather I'd suggest to make a CLNE-MATEKF411 configuration. Then we also can add ICM20602 as used by Eachine Tyro79 (see https://github.com/betaflight/unified-targets/issues/1119#issuecomment-1683989184)

---

I believe this board differs from the MATEKF411 target with the addition of flash on SPI2, as well as different timer assignments. Not sure if it would work for the Tyro79 FC.

---

Fwiw, physically it looks like no Marek board I was able to find googling around. And won't people search under "F" since F...reeAxis is what they bought it as?

---

Cannot spot the timer difference. This PR only changes ACC and GYRO defines compared to the original MATEKF411 config and adds FLASH driver and setting: `flash_spi_bus = 2` which would be shared with OSD. To make it worse the flash chip is using W25N01G.

@SteveCEvans what do you think?

---

@ot0tot please only change `manufacturer_id CLNE`

---

Made some additional changes to the config